ci/jenkins/test.sh

```diff
@@ -565,22 +565,30 @@ function deliver_antrea_linux_containerd {
     echo "==== Start building and delivering Linux containerd images ===="
     DOCKER_REGISTRY="${DOCKER_REGISTRY}" ./hack/build-antrea-linux-all.sh --pull
     docker save -o antrea-ubuntu.tar antrea/antrea-ubuntu:latest
-    # containerd is the runtime, need to load the image via ctr.
-    ctr -n=k8s.io images import antrea-ubuntu.tar
     echo "===== Pull necessary images on Control-Plane node ====="
     harbor_images=("agnhost:2.13" "nginx:1.15-alpine")
     antrea_images=("e2eteam/agnhost:2.13" "docker.io/library/nginx:1.15-alpine")
     common_images=("registry.k8s.io/e2e-test-images/agnhost:2.29")
     k8s_images=("registry.k8s.io/e2e-test-images/agnhost:2.45" "registry.k8s.io/e2e-test-images/jessie-dnsutils:1.5" "registry.k8s.io/e2e-test-images/nginx:1.14-2")
     e2e_images=("k8sprow.azurecr.io/kubernetes-e2e-test-images/agnhost:2.45" "k8sprow.azurecr.io/kubernetes-e2e-test-images/jessie-dnsutils:1.5" "k8sprow.azurecr.io/kubernetes-e2e-test-images/nginx:1.14-2")
 
-    for i in "${!harbor_images[@]}"; do
-        ctr -n=k8s.io images delete "${antrea_images[i]}"
-        ctr -n=k8s.io images pull "${DOCKER_REGISTRY}/antrea/${harbor_images[i]}"
-        ctr -n=k8s.io images tag "${DOCKER_REGISTRY}/antrea/${harbor_images[i]}" "${antrea_images[i]}"
+    echo "===== Deliver Antrea YAML to Controller nodes ====="
+    IP=$(kubectl get nodes -o wide --no-headers=true | awk -v role="$CONTROL_PLANE_NODE_ROLE" '$3 ~ role {print $6}')
+    HOST_IPS=$(ip addr show | grep -oP 'inet \K[\d.]+')
+    matched=false
+    for host_ip in $HOST_IPS; do
+        if [[ $HOST_IP == $IP ]]; then
```

Comment:
I think bash variable is case-sensetive so this is not going to work.
But did you consider this comment?
> But since you didn't check whether this is the jumper node itself when delivering the image, perhaps no need to check if it's itself when delivering the yaml too?